antlr4/PostgreSQLParser.g4

```diff
@@ -81,8 +81,12 @@ stmt
      | truncate_stmt
      | unlisten_stmt
      | update_stmt
-     | vacuum_stmt
-     | values_stmt)
+     | vacuum_stmt)
+    ;
+
+data_stmt
```

Comment:
I see your point of view here, but I would like to make this list of statements consistent with the ones documented: https://www.postgresql.org/docs/10/sql-commands.html

Since the TABLE statement is considered a type of SELECT statement according to Postgres, I feel that's where the TABLE statement belongs, and based on the documentation link you pointed to, it should be as simple as adding `| TABLE ONLY? table_name_ STAR?` somewhere in `select_stmt` or we can retain the `table_stmt` rule you added and nest that under `select_stmt`. The test cases you correctly identified and added (👍) should all pass this form.

The `data_stmt` rule should not exist. `select_stmt` and `values_stmt` should still be under `stmt`.

Reply:
You are right. `stmt` could be consistent with documented ones.
I has fixed.
Thank you!